script/playbackstrategy/msestrategy.js

```diff
@@ -68,10 +71,16 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
 
       function onBuffering () {
         isEnded = false;
-        publishMediaState(MediaState.WAITING);
+        if (isSeeking && !publishedSeekEvent) {
+          publishMediaState(MediaState.WAITING);
+          publishedSeekEvent = true;
+        } else if (!isSeeking) {
+          publishMediaState(MediaState.WAITING);
+        }
```

Comment:
This would work too, but is arguably less readable.
```
if (!isSeeking || !publishedSeekEvent) {
  publishMediaState(MediaState.WAITING);
  publishedSeekEvent = true;
}
```